rpki-validator/src/main/resources/packaging/generic/bin/rpki-validator-3.sh

```diff
@@ -64,10 +64,13 @@ CONFIG_DEFAULT_FILE="${CONFIG_DIR}/application-defaults.properties"
 
 function parse_config_line {
     local CONFIG_KEY="$1"
```

Comment:
Here is the main change, order of looking up standard up first and fall back to default.